bindings/java/src/main/java/io/github/treesitter/jtreesitter/java/TreeSitterJava.java

```diff
@@ -0,0 +1,38 @@
+package io.github.treesitter.jtreesitter.java;
+
+import java.lang.foreign.*;
+import java.lang.invoke.MethodHandle;
+
+/** Java grammar for tree-sitter */
+public final class TreeSitterJava {
+    private static final ValueLayout VOID_PTR =
+            ValueLayout.ADDRESS.withTargetLayout(MemoryLayout.sequenceLayout(Long.MAX_VALUE, ValueLayout.JAVA_BYTE));
+    private static final FunctionDescriptor FUNC_DESC = FunctionDescriptor.of(VOID_PTR);
+    private static final Linker LINKER = Linker.nativeLinker();
+    private static final TreeSitterJava INSTANCE = new TreeSitterJava();
+
+    private final Arena arena = Arena.ofAuto();
+    private final String library = System.mapLibraryName("tree-sitter-java");
+    private final SymbolLookup symbols =
+            SymbolLookup.libraryLookup(library, arena).or(SymbolLookup.loaderLookup());
```

Comment:
Is this `or(SymbolLookup.loaderLookup())` actually needed? If I understand it correctly, the library loaded by `SymbolLookup#libraryLookup` is expected to contain the `tree_sitter_java` function.

And if `SymbolLookup#libraryLookup` fails to load the library, it already throws an exception. So I am wondering if `SymbolLookup.loaderLookup()` will actually ever be used here.